Removed LICENSE.*, NOTICE, PROCESS files

Codysseus opened the pull request:
Please let me know if there are more of these files to remove :)

rminnich commented:
I think this is close! Thanks! 
However, I looked at what 9front did and realized we can't quite get rid of the GPL and AFPL: 
```
 Plan 9 from Bell Labs is provided under the terms of the MIT license,
reproduced in the file /lib/legal/mit.

Any additions or changes (as recorded in git history) made by 9front
are also provided under the same MIT License, unless otherwise indicated.

The following exceptions apply:

1. No right is granted to create derivative works of or
   to redistribute (other than with the Plan 9 Operating System)
   the screen imprinter fonts identified in subdirectory
   /lib/font/bit/lucida and printer fonts (Lucida Sans Unicode, Lucida
   Sans Italic, Lucida Sans Demibold, Lucida Typewriter, Lucida Sans
   Typewriter83), identified in subdirectory /sys/lib/postscript/font.
   These directories contain material copyrights by B&H Inc. and Y&Y Inc.

2. The printer fonts identified in subdirectory /sys/lib/ghostscript/font
   are subject to the GNU GPL, version 2, reproduced in the file /lib/legal/gpl.

3. The ghostscript program in the subdirectory /sys/src/cmd/gs is
   covered by the Aladdin Free Public License, reproduced in the file
   /lib/legal/afpl.

4. The Source and Terminus fonts identified in subdirectory
   /lib/font/bit/ are released under the SIL Open Font License,
   reproduced in the file /lib/legal/ofl.

Other, less notable exceptions are marked in the file tree with
COPYING, COPYRIGHT, or LICENSE files.
```
So, I would suggest, as part of this, just 
mkdir -p lib/legal
and put the licenses in there, add the bit, and add this notice.

That ought to be enough for the time we have left. 

OR

leave the GPL and AFPL files in, update NOTICE, and add mit. 

Thanks very much!

Codysseus commented:
I went with the directory method as specified by the notice (and as used by 9front).

Though I am a bit clueless on if I need to change any language. I added a line to the MIT license specifying NIX authors, but I'm not sure if I need to change this notice and/or the MIT license as I also included the line from 9front (since we'll be building on top of that).

Thank you for your encouragement and help :)

Codysseus commented:
Updated my fork, removed unnecessary licenses for files that don't exist. The only question now is if the attribution of the MIT license it correct :)
